codersdk/insights.go

```diff
@@ -98,7 +98,8 @@ type UserActivity struct {
 // ConnectionLatency shows the latency for a connection.
 type ConnectionLatency struct {
 	P50 float64 `json:"p50" example:"31.312"`
-	P95 float64 `json:"p95" example:"119.832"`
```

Comment:
Is this change intentional?

Reply:
It was, but I reverted it. (It was based on a discussion in Slack where p90/p99 would be preferable over p95, and since this p95 isn't used anywhere, the change would be harmless, but this is not the right place for it.)